wiki/People/The_Team/Project_Loved_Team/ru.md

```diff
@@ -15,7 +16,7 @@ outdated: true
 
 ## Члены команды
 
-![][flag_US] **[Toy](https://osu.ppy.sh/users/2757689)** управляет Project Loved и координирует большинство действий команды.
+![][flag_US] **[clayton](https://osu.ppy.sh/users/3666350)** курирует проект, пишет новости и посты на форуме. ![][flag_LT] [huu](https://osu.ppy.sh/users/6044237) ведёт общедоступные таблицы и помогает модерировать обсуждения о категории «Любимые», а ![][flag_US] [Noffy](https://osu.ppy.sh/users/1541323) проверяет метаданные карт, которые будут выставлены на голосование. Остальная часть команды — это капитаны:
```

Comment:
I've updated this part in English one more time (since I guess we're doing both in this PR now), so it needs to be updated here again.